cmd/service_windows.go

```diff
@@ -124,7 +124,7 @@ var mountServiceCmd = &cobra.Command{
 
 		err = mountInstance()
 		if err != nil {
-			return fmt.Errorf("failed to mount instance [%s]", err.Error())
+			return fmt.Errorf("failed to mount instance [%s]\n Verify WinFSP is installed correctly", err.Error())
```

Comment:
Does this error only occur if WinFSP is not installed? Or could there be another reason. I would just want to try and isolate the error for WinFSP so that users don't feel frustrated needing to verify their WinFSP installation when the issue could be something else.

Reply:
This error is added in place without checking if WinFSP is already installed. I see Michael's comment about the push he put in place on where the specific error can be output. I'll use his implementation

Reply:
Yeah I think that fix would work well.